README.md

```diff
@@ -50,6 +50,29 @@ relied upon.
 
 When using an emulator, you can press `cmd+m` to simulate a menu button press.
 
+### Building with API keys
+Certain services require an API key, so you'll need to build with the key to use them in the apk.
+
+1. To build with the API key (for services such as Sentry), add a `<project-dir>/.<service>_release`
```

Comment:
nit: -> `.<service>_debug` We should discourage people from using release builds because it will skew our data.